minimal/modules/dnpm-node-compose.yml

```
@@ -74,9 +74,27 @@ services:
         condition: service_healthy
     labels:
       - "traefik.enable=true"
-      - "traefik.http.routers.dnpm-backend.rule=PathPrefix(`/api`)"
       - "traefik.http.services.dnpm-backend.loadbalancer.server.port=9000"
+      # expose everything
+      - "traefik.http.routers.dnpm-backend.rule=PathPrefix(`/api`)"
       - "traefik.http.routers.dnpm-backend.tls=true"
+      - "traefik.http.routers.dnpm-backend.service=dnpm-backend"
+      # except ETL
+      - "traefik.http.routers.dnpm-backend-etl.rule=PathRegexp(`^/api(/.*)?etl(/.*)?$`)"
+      - "traefik.http.routers.dnpm-backend-etl.tls=true"
+      - "traefik.http.routers.dnpm-backend-etl.service=dnpm-backend"
+      - "traefik.http.routers.dnpm-backend-etl.middlewares=dnpm-backend-etl"
+      # create this with "echo $(htpasswd -nB USER) | sed -e s/\\\$/\\\$\\\$/g"
+      # this needs an ETL processor with support for basic auth
+      - "traefik.http.middlewares.dnpm-backend-etl.basicauth.users=${ETL_PASSWD}"
```

We have a correctly formatted basic auth user pw generation already setup globally. The credentials for the user are automatically stored in `/etc/bridghead/$PROJECT.local.conf` (git ignored).
```suggestion
      - "traefik.http.middlewares.dnpm-backend-etl.basicauth.users=${LDM_AUTH}"
```

---

Thanks! I've used the existing `auth` middleware as some other containers seem to do.